2018/spring/Sterne-MW@3pm/qifan-yang.md

```diff
@@ -0,0 +1,4 @@
+# Bibliography
+This is *Qifan Yang*. He has been living in this world for **7738** days! [^fn1]
+
+[^fn1]: Hello World!
```

Comment:
Are you trying to use a Markdown feature here? If so, it isn't working. :)

Reply:
[^fn1] is supposed to be a footnote (which work on online markdown editors).
Maybe it doesn't work here :(